twine/commands/check.py

```diff
@@ -97,18 +103,7 @@ def _check_file(filename, render_warning_stream):
     return warnings, is_ok
 
 
-# TODO: Replace with textwrap.indent when Python 2 support is dropped
-def _indented(text, prefix):
```

Comment:
This muddies the review by having ancillary work in the pull request. Please avoid this in the future

Reply:
Appreciated, though in my defense I removed it because the internal function definition was failing type-checking, and it seemed expedient to address both issues at once.